app/config.py

```diff
@@ -0,0 +1,17 @@
+"""Flask configuration."""
+import os
+
+"""
+Application Config
+"""
+
+SESSION_COOKIE_NAME = os.environ.get("SESSION_COOKIE_NAME") or "session_cookie"
```

Comment:
:recycle: Is there a way to reduce the code duplication in this file?

Reply:
Suggestion?

Reply:
🤔 There's a few things one could do. 

I'd recommend looking at how large amounts of .env variables are used in some well established projects. Typically .env files are used to setting large amounts of environment variables, but in our case we are using GitHub secrets - so this is not applicable. But just have a think about if you think this method of setting envs is a sustainable practice.

:pushpin: A big problem with this approach which worries me is that large amounts of the program could run with test envs used, suppose if an env is not set so a default is used.

For example. we deploy this project and a test value is used and so user data is lost because its being sent to a dictionary rather than an sql db. We wouldn't know until the python process dies and the data is lost. There's no warning if a default value (typically a junk value) for the envs is used. 

@harryyo What do you think? I'm I worried about nothing? 

If I'm right to be worried then the solution is to write a basic function which is basically a decorated version of the get function which prints a warning if a default value is set.

Reply:
Thank you so much both of you. So I understand properly when we have a huddle tomorrow, is it ok to ask a few questions?

Reply:
We're about to implement monitoring and alerting - we can raise an automated alert if this appears in logs @vialde

Reply:
Perhaps we park this for now though and create a new ticket for it as is not specific to this repo?